bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -190,6 +190,11 @@ public String getId() {
         }
     }
 
+    @Override
+    public String getName() {
+        return getCurrentPage() != null ? getCurrentPage().getPath() : getPath();
```

Comment:
What is this getName and why is name returning a path ? When form container is used inside sites, page component anyways returns a path

Reply:
What is the use-case to return page path inside form container